js/src/ops.ts

```diff
@@ -152,3 +156,15 @@ function encodeVarintProto(n: number): Uint8Array {
   enc = [...enc, l];
   return new Uint8Array(enc);
 }
+
+function encodeFixed32LE(n: number): Uint8Array {
+  const enc = new Uint8Array(4);
+  let l = n;
+  for (let i = enc.length; i > 0; i--) {
+    /* tslint:disable */
+    enc[Math.abs(i - enc.length)] = l;
```

Comment:
I would do ` = (l % 256)`. That is quite clear what you want. I think this will happen automatically in JS, but it is harder to review and ensure correctness.

Reply:
That wouldn't actually work correctly as the bits need to be truncated and using modulo would give you incorrect results. The most correct thing would actually be `l & 0xFF` but I avoided that because the linter didn't like it. But since we're shutting off the linter for mutation anyway I think we can add this to make it more clear.

Reply:
Yeah makes sense, I prefer bitwise ops (e.g., `l & 0xFF`), but the linter doesn't like them for some reason.

Reply:
Changed.